SoftLayer/CLI/vpn/ipsec/order.py

```diff
@@ -0,0 +1,29 @@
+"""Order a IPSec VPN tunnel."""
+# :licenses: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.option('--datacenter', '-d', required=True, prompt=True, help="Datacenter shortname")
+@environment.pass_env
+def cli(env, datacenter):
+    """Order/create a IPSec VPN tunnel instance."""
+
+    ipsec_manager = SoftLayer.IPSECManager(env.client)
+
+    result = ipsec_manager.order(datacenter, ['IPSEC_STANDARD'])
```

Comment:
Like ipsec has a cost, I think it is a good idea to add a confirmation before to order
```
        if not (env.skip_confirmations or formatting.confirm(
                "This action will incur charges on your account. Continue?")):
            raise exceptions.CLIAbort('Aborting ipsec order.')
```